app/views/shared/_callout.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-md-6">
+    <div class="bs-callout bs-callout-danger">
+      <h3><i class="fa fa-exclamation-circle" aria-hidden="true"></i> Notice</h3>
+      <p>Returned materials may be quarantined for four days and will remain on your account during the quarantine period.</p>
```

Comment:
can the message added to en.yml?

Reply:
I guess 🤷 

Sometimes I feel like adding the i18n component is kind of a waste of time because we literally only have english. I'll move it though that's fine.

Reply:
thanks, I think it might make this partial look more generic